pkg/grpc/metadata/metadata.go

```diff
@@ -0,0 +1,16 @@
+package grpc_metadata
```

Comment:
Put this file into pkg/grpcutils/metadata.go and name the package grpcutils, so that:
- package has the same name as the directory
- there are no unnecessary directories (grpc that does not contain any go files)

Reply:
However this is consistent with `pkg/http/...` stuff. What do you think?

Reply:
It's not clear whether we will have more than one function in that package, so I'd opt for simplicity. If we have a lot of files/lot of functionality in grpcutils package (or some other name), we can split it into multiple packages.